### DISCUSSION TITLE
How to print the current character encoding?

### DISCUSSION THREAD

**Discussion by CN-GUOYIMING** [nushell/nushell#13989] [Q&A] [answered]
I wrote some Japanese Git commit messages, and when I used `git log`, they displayed correctly in the terminal. However, they became garbled when I executed `git log | clip` and pasted them into a UTF-8 Markdown file.

How can I display current character encoding in Nusehll?

I found a solution in issue #8117. However, I still don't know how to check it. I couldn't find useful information in the Nushell documentation.

My Nushell version is 0.98.0.

**Comment by fdncred** [comment, marked as answer]
If you're using windows, you use `chcp` to tell what character code page your terminal is set to. I believe nushell itself wants things to be in utf-8 since that's what rust wants.

**Reply by CN-GUOYIMING** [reply]
Oh, it worked! Thank you!
